README.md

```diff
@@ -110,21 +110,20 @@ curl -X POST 'http://localhost:8081' \
 }'
 ```
 
-## Convenience Functionality 🧰
+## Working with `falconpy`
 
-### `falconpy`
+As of version 1.0.0, `falconpy` no longer ships with Foundry Function Python.
```

Comment:
nit: drop the first two lines here. No one knows we had `falconpy` in this repo as the number of users is quite low. Just remove the noise and get straight to the point below. Run the risk of confusing folks here with to much context.